sagemaker-inference-recommender/xgboost/xgboost-inference-recommender.ipynb

```diff
@@ -0,0 +1,793 @@
+{
```

Comment:
<pre class="ql-syntax" spellcheck="false">Line #4.    stopped = False
</pre><p>Nit - "stopped" variable name is a bit misleading since it represents the statuses of completed, stopped, and failed. Could rename to "finished" or similar?</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/aws/amazon-sagemaker-examples/pull/3073/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='49'/>

Reply:
<p>Done</p>